templates/test/tr_job_result_details.html.ep

```diff
@@ -0,0 +1,7 @@
+% if ($comments and $comments > 0) {
```

Comment:
this code section is included in two locations, templates/test/previous.html.ep with the _resultset_ including comments as parameter (always defined but can be empty) and templates/test/tr_job_result.html.ep with _$res->{comments}_ as parameter which is a scalar (undef if no comments).
If I just check with `if ($comments)` then it will always be true for "previous.html" and always display the symbol. If I just check with `if ($comments > 0)` then it will yield a warning for "tr_job_result.html".

So what approach do you prefer?